src/styles/variables.sass

```diff
@@ -0,0 +1,84 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
+$em-base: $base-font-size
+
+// Colors
+$c-main: green
+$c-comp: #333
+$c-alt: #000
+
+$text-color: $c-comp
+$text-color-light: #676767
+$text-back: #FFF
+
+// Breakpoints
+$break320: '(min-width: 320px)'
+$break480: '(min-width: 480px)'
+$break768: '(min-width: 768px)'
+$break992: '(min-width: 992px)'
+$break1200: '(min-width: 1200px)'
+
+$break320M: '(max-width: 319px)'
+$break480M: '(max-width: 479px)'
+$break768M: '(max-width: 767px)'
+$break992M: '(max-width: 991px)'
+$break1200M: '(max-width: 1199px)'
+
+$landscape: '(orientation: landscape)'
+$portrait: '(orientation: portrait)'
+
+// Mixins
+
+// +break(768) => @media (min-width: 768px)
+=break($min)
+	@media (min-width: #{$min}px)
+		@content
+
+// +media($break320M) => @media (max-width: 319px)
+=media($var)
+	@media #{$var}
+		@content
```

Comment:
Tohle byla co nejjednodušší rychlovka... Codepen vypadá fajn, co k tomu přidat nějakou dokumentaci, ať to umí pak každej používat? :)